CDP4IMEInstaller/CDP4IME.wxs

```diff
@@ -44,167 +44,167 @@
       <!-- DevExpress Dependencies -->
 
       <Component Id="_A726CFBE26D5" Guid="{90BBA601-EC72-48D0-B370-A726CFBE26D5}" Win64="$(var.Win64)">

```

Comment:
Under the impression that in devexpress 19.2 the number of required files and some names have changed. This seems incomplete.